seagence/manifest.json

```diff
@@ -0,0 +1,61 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "94f4e504-c98c-466f-b934-5e5ee0331944",
+  "app_id": "seagence",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Defect Monitoring and Root Cause Automation using Seagence",
+    "title": "Seagence",
+    "media": [
+      {
+        "media_type": "video",
+        "caption": "Seagence defect monitoring explainer video",
+        "image_url": "images/thumbnail.png",
+        "vimeo_id": 123456789
+      },
+      {
+        "media_type": "image",
+        "caption": "Defect and Success Clusters",
+        "image_url": "images/defect-and-success-clusters.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Success and Defect Execution Paths",
+        "image_url": "images/defect-and-successexecution-paths-1440x810.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
```

Comment:
Please include relevant categories. Additionally, if your integration is sending data into Datadog, please include a Submitted Data Type as seen [here](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags). If you are pulling out data, please include Queried Data Type.